aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/test_instrumentation_patch.py

```diff
@@ -229,10 +325,70 @@ def _do_extract_sqs_attributes() -> Dict[str, str]:
     return _do_extract_attributes(service_name, params)
 
 
-def _do_extract_attributes(service_name: str, params: Dict[str, str]) -> Dict[str, str]:
+def _do_extract_attributes_bedrock(service, operation=None) -> Dict[str, str]:
+    params: Dict[str, Any] = {
+        "agentId": _BEDROCK_AGENT_ID,
+        "dataSourceId": _BEDROCK_DATASOURCE_ID,
+        "knowledgeBaseId": _BEDROCK_KNOWLEDGEBASE_ID,
+        "guardrailId": _BEDROCK_GUARDRAIL_ID,
+        "modelId": _GEN_AI_REQUEST_MODEL,
+    }
+    return _do_extract_attributes(service, params, operation)
+
+
+def _do_on_success_bedrock(service, operation=None) -> Dict[str, str]:
+    result: Dict[str, Any] = {
+        "agentId": _BEDROCK_AGENT_ID,
+        "dataSourceId": _BEDROCK_DATASOURCE_ID,
+        "knowledgeBaseId": _BEDROCK_KNOWLEDGEBASE_ID,
+        "guardrailId": _BEDROCK_GUARDRAIL_ID,
+        "modelId": _GEN_AI_REQUEST_MODEL,
+    }
+    return _do_on_success(service, result, operation)
+
+
+def _do_extract_bedrock_runtime_attributes() -> Dict[str, str]:
```

Comment:
Not blocking: This is the same as _do_extract_attributes_bedrock, can be deprecated

Reply:
Deprecated _do_extract_bedrock_runtime_attributes, and use _do_extract_attributes_bedrock